docs/getting-started.md

```diff
@@ -65,11 +65,39 @@ This sends telemetry to Microsoft by default. You can prevent the telemetry from
+| `--emulator`          | boolean    | Deploys the app to an emulator                   |
+| `--device`            | boolean    | Deploys the app to a connected device            |
+| `--target`            | string     | Deploys the app to the specified GUID for a device |
+| `--remote-debugging`  | boolean    | Deploys the app in remote debugging mode.        |
+| `--logging`           | boolean    | Enables logging                                  |
+| `--no-packager`       | boolean    | Do not launch packager while building            |
+| `--bundle`            | boolean    | Enable Bundle configuration and it would be ReleaseBundle/DebugBundle other than Release/Debug |
+| `--no-launch`         | boolean    | Do not launch the app after deployment           |
+| `--no-autolink`       | boolean    | Do not run autolinking                           |
+| `--no-build`          | boolean    | Do not build the solution                        |
+| `--no-deploy`         | boolean    | Do not deploy the app                            |
+| `--deploy-from-layout`| boolean    | Force deploy from layout, even in release builds |
+| `--sln`               | string     | Override the app solution file determined by 'react-native config', e.g. windows\myApp.sln |
+| `--proj`              | string     | Override the app project file determined by 'react-native config', e.g. windows\myApp\myApp.vcxproj |
+| `--msbuildprops`      | string     | Comma separated props to pass to MSBuild, eg: prop1=value1,prop2=value2 |
+| `--buildLogDirectory` | string     | Optional directory where MSBuild log files should be stored |
+| `--info`              | boolean    | Dump environment information                     |
+| `--direct-debugging`  | number     | Enable direct debugging on specified port        |
+| `--useWinUI3`         | boolean    | Targets WinUI 3.0 (Preview) instead of UWP XAML. |
+| `--useHermes`         | boolean    | Use Hermes instead of Chakra as the JS engine (supported on 0.64+) |
+| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |
+| `-h`, `--help`        | boolean    | output usage information                         |
+
   A new Command Prompt window will open with the React packager as well as a `react-native-windows` app. This step may take a while during first run since it involves building the entire project and all dependencies. You can now start developing! :tada:
 
-  This sends telemetry to Microsoft by default. You can prevent the telemetry from being sent by using the `--no-telemetry` command line option. See the `@react-native-windows/cli` README for more details.
-  
-  For a description of the options that `run-windows` offers, see [run-windows usage](https://github.com/microsoft/react-native-windows/blob/main/packages/@react-native-windows/cli/README.md#usage).

```

Comment:
The plan is we keep these both up to date in duplicate? Do we have comments in the code to help guide people to what they need to update when they add/remove/change a CLI option? Is there anything more we expect the CLI readme will have that would justify keeping a back link here, or is it meant to be a subset?

Reply:
The CLI readme has to continue to exist because when the @react-native-windows/cli package is published to npm the README in that directory is used as the package description in npm. Information on CLI options is consistent across the README and what I'm adding to the website, only different is that the README has a longer blurb about the telemetry the package captures.

Reply:
While I like moving some of this into the docs, I think it makes more sense to create a new page and not dump this giant table in the Getting Started doc.

Reply:
Yeah, that table dump for init options would probably be better put a link away, i.e. "For more details on the options for react-native-windows-init, see XYZ"

Reply:
@jonthysell @chrisglein Is there a preference for how we want to organize the info? Should the link just take people back to the README or to the NPM page that contains the README info and well as versioning info, etc., or should it take people to another page on the website with the options laid out?

Reply:
Because the CLI options change between versions, the link should go to a page with the correct table for that version. That could be the right release readme on NPM, though it might be easier to just have those pages within our docs and its versioning system.

As for the readme's themselves, as the CLI commands aren't really "standalone" (you only use them within the context of creating/building a RNW project) I'd be fine with reducing the readme to more-or-less a link back to our docs. That's what the normal CLI React Native does:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/10852185/149420241-3279c038-58a4-4c76-a02a-dab2cb543412.png">

Reply:
Ok @jonthysell just update PR with those changes whenever you have a chance to review.